app/main.py

```diff
@@ -1,8 +1,43 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str):
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: float,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int):
+
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list[Car]):
+        price = 0
+        for car in cars:
+            if self.clean_power > car.clean_mark:
+                price += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+        return price
+
+    def calculate_washing_price(self, car: Car):
+        return round(
+            car.comfort_class
+            * (self.clean_power - car.clean_mark)
+            * self.average_rating / self.distance_from_city_center,
+            1)
+
+    def wash_single_car(self, car: Car):
+        car.clean_mark = self.clean_power
```

Comment:
You need to check if `clean_power` is greater than `clean_mark` here as well because in case you run this method separately from others it will not work correctly

Reply:
Fixed, also added annotation for return in methods